pkg/version/types.go

```diff
@@ -16,21 +16,61 @@ limitations under the License.
 
 package version
 
-import "fmt"
+import (
+	"fmt"
+)
 
 // Info contains versioning information.
 // TODO: Add []string of api versions supported? It's still unclear
 // how we'll want to distribute that information.
 type Info struct {
-	GitVersion string `json:"gitVersion"`
-	GitCommit  string `json:"gitCommit,omitempty"`
-	BuildDate  string `json:"buildDate"`
-	GoVersion  string `json:"goVersion"`
-	Compiler   string `json:"compiler"`
-	Platform   string `json:"platform"`
+	GitVersion string `json:"gitVersion" yaml:"gitVersion"`
+	GitCommit  string `json:"gitCommit,omitempty" yaml:"gitCommit,omitempty"`
+	BuildDate  string `json:"buildDate" yaml:"buildDate"`
+	GoVersion  string `json:"goVersion" yaml:"goVersion"`
+	Compiler   string `json:"compiler" yaml:"compiler"`
+	Platform   string `json:"platform" yaml:"platform"`
```

Comment:
why add yaml?

Reply:
output can use “-o yaml | json” to define output format

Reply:
But in k8s crd, we can '-o yaml' to format output too, and we don't re-define `yaml` filed, what's the difference?

Reply:
no difference ,just for format convert convience.
another point is crictl dont have -o options